docs/fr/docs/advanced/response-directly.md

```diff
@@ -1,63 +1,63 @@
-# Return a Response Directly
+# Retourner une réponse directement
```

Comment:
For me the sentence `Renvoyer directement une réponse` sounds better.

Reply:
I correct this, if it's ok for you, please, close the comment. Thanks.